.poetry/bin/poetry

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
```

Comment:
This `.poetry` folder shouldn't be in the codebase. It should just be installed on your system.